valkyrie.drush.inc

```diff
@@ -346,6 +350,20 @@ function drush_valkyrie_update() {
     }
   }
 }
+function drush_valkyrie_upgrade() {
+  $current_path = getcwd();
+  $src = dirname(__FILE__);
+  if($src != $current_path) {
+    drush_print('You need to be in (~/.drush/valkyrie) to do this');
+  }
+  else {
+    drush_print('Checking for updates...');
```

Comment:
There are a number of things that we might want to do here, in addition. For example, ensure that we're on the correct branch. We might also want to add `--rebase`, gracefully handle errors, etc.

I'd suggest that we use `drush_shell_exec()` rather the `drush_shell_proc_open()` by default. Maybe check `drush_get_option('debug', FALSE)` to display the full output.